lib/jekyll/converters/markdown.rb

```diff
@@ -15,7 +15,7 @@ def setup
       rescue NameError
         Jekyll.logger.error "Invalid Markdown Processor:", "#{@config['markdown']}"
         Jekyll.logger.error "", "Valid options are [ #{PARSERS.keys.join(" | ")} ]"
-        raise Jekyll::FatalException, "Invalid Markdown Processor: #{@config['markdown']}"
+        raise Jekyll::Errors::FatalException, "Invalid Markdown Processor: #{@config['markdown']}"
```

Comment:
Line is too long. [98/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.